.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+
+
+Our tests check whether you have introduced trailing white space. If such a test fails, please check the "artifacts button" above, which if you click it gives a link to a downloadable file to help you identify the issue. One can also run scripts/remove_trailing_whitespace.sh locally if you have a bash shell.
```

Comment:
Once can also -> You can also

bash shell. -> bash shell and sed available.

Reply:
Fixed.